distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -767,6 +771,102 @@ type Table
 
+    ## ALIAS append, concat
+       Appends records from other table(s) to this table.
+
+       Arguments:
+       - tables: A single table or a vector of tables to append to this one. The
+         tables are concatenated in the order they are specified, with `self`
+         being the first one.
+       - match_columns: Specifies how to match the columns.
+         - If `Match_Columns.By_Name` - the columns are matched by name across
+           all provided tables.
+           If unmatched columns are to be dropped, the resulting table will keep
+           only the set of columns that appear in all provided tables, in the
+           relative order that they appeared in the `self` table.
+           If unmatched columns are kept, they are added in the order of
+           appearance - i.e. first all columns from `self` will be added in the
+           original order, then any columns from the second table that were not
+           matched will be added at the end (preserving their relative order),
+           and so on for all the remaining tables.
+         - If `Match_Columns.By_Position` - the columns are mapped by position.
+           If unmatched columns are to be dropped, the resulting table will have
+           as many columns as the table that had the least columns and the
+           column names of the first table (self) will be used.
+           If unmatched columns are kept, the resulting table will have as many
+           columns as the table with the most columns. Since the first table may
+           not have all the necessary columns to provide column names for the
+           result, the result will have column names taken from the first table
+           that has the biggest number of columns.
+       - keep_unmatched_columns: If set to `True`, unmatched columns are kept
+         and are padded with `Nothing` for tables that did not have them.
+         If set to `False`, only the common subset of columns is kept - any
+         column that is not present in all tables is dropped. Defaults to
+         `Report_Unmatched`, which behaves like `True` - unmatched columns are
+         kept and padded with `Nothing`, but a problem is reported.
+       - allow_type_widening: Specifies if the resulting column type should be
+         adapted to fit columns from all arguments; defaults to `True`. If set
+         to `True` a common type will be chosen for each column, as described in
+         the section "Unifying Column Types" below. If set to `False` the
+         resulting column type will be the same as in the first table the
+         corresponding column appeared in (usually `self`, except for unmatched
+         columns). If type widening is not allowed, all columns that are merged
+         with each other must have the same type as the first one. The only
+         exception is if the first column had `Mixed` type which will accept any
+         other type.
```

Comment:
```suggestion
       - allow_type_widening: Specifies if the resulting column type should be
         adjusted to fit columns from all arguments. If `True`, a common type 
         will be chosen for each column (see "Unifying Column Types" below).
         If `False`, the resulting column type will be the same as in the first
         table containing the column. In this case, all columns that are
         concatenated must have the same type as the first one (unless this
         had a `Mixed` type - in which case it will accept any other types).
```